library/std/src/fs.rs

```diff
@@ -408,6 +408,7 @@ impl File {
     ///
     /// This function will create a file if it does not exist, or return an error if it does. This
     /// way, if the call succeeds, the file returned is guaranteed to be new.
+    /// If a file exists at the target location before, creating file will fail with [`AlreadyExists`].
```

Comment:
Tense mixup. It should probably suggest people look at `OpenOptions::open` which explains the possible errors in more detail.
```suggestion
    /// If a file exists at the target location, creating a new file will fail with [`AlreadyExists`]
    /// or another error based on the situation. See [`OpenOptions::open`] for a
    /// non-exhaustive list of likely errors.
```